src/Util/TemplateLinter.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Component\Process\Process;
+
+/**
+ * Linters used by make:* commands to cleanup the generated files.
+ *
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class TemplateLinter
+{
+    private bool $usingBundledPhpCsFixer = true;
+    private bool $usingBundledPhpCsFixerConfig = true;
+
+    public function __construct(
+        private ?string $phpCsFixerBinaryPath = null,
+        private ?string $phpCsFixerConfigPath = null,
+    ) {
+        $this->setBinary();
+        $this->setConfig();
+    }
+
+    public function lintFiles(array $templateFilePaths): void
+    {
+        $phpFiles = [];
+
+        foreach ($templateFilePaths as $filePath) {
+            if (str_ends_with($filePath, '.php')) {
+                $phpFiles[] = $filePath;
+            }
+        }
+
+        $this->lintPhpTemplate($phpFiles);
+    }
+
+    public function lintPhpTemplate(string|array $templateFilePath): void
+    {
+        if (\is_string($templateFilePath)) {
+            $templateFilePath = [$templateFilePath];
+        }
+
+        foreach ($templateFilePath as $filePath) {
+            $process = Process::fromShellCommandline(sprintf('php %s --config=%s --using-cache=no fix %s', $this->phpCsFixerBinaryPath, $this->phpCsFixerConfigPath, $filePath));
+
+            $process->run();
+        }
+    }
+
+    public function getLinterUserMessage(): array
+    {
+        $message = [];
+        $message[0] = 'Linting Generated Files With:'.\PHP_EOL;
```

Comment:
Do we need the EOL? I think if we pass this to `$output->writeln()` then each item in the array becomes a new line.